portal/patient_self_checkin/index.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Patient Self Check in
+ *
+ * This program allows patients to check themselves in using a touchscreen or similar device in the doctors' practice.
+ * Doing so, they are marked as arrived (code @) on the calendar module of the clinician's screen and are also marked
+ * as present in the Patient Flow Board.
+ * The purpose of this program is to free up time for front desk staff.
+ *
+ * @category  Portal
+ * @package   OpenEMR
+ * @author    Alfie Carlisle <asc@carlisles.co>
+ * @copyright Copyright (c) 2017 Alfie Carlisle <asc@carlisles.co>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @link      http://www.open-emr.org
+ */
+
+$ignoreAuth = true;
```

Comment:
globals.php does all the authentication(and the $ignoreAuth flag prior to it causes it to skip authentication). Header brings in front end libraries.